src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -1113,53 +1114,57 @@ private static int getTableBorder(AttributeSet tableAttr) {
-                                 new CSS.Attribute[]{CSS.Attribute.BACKGROUND_COLOR});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.WIDTH,
-                                 new CSS.Attribute[]{CSS.Attribute.WIDTH});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.HEIGHT,
-                                 new CSS.Attribute[]{CSS.Attribute.HEIGHT});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.BORDER,
-                                 new CSS.Attribute[]{CSS.Attribute.BORDER_TOP_WIDTH, CSS.Attribute.BORDER_RIGHT_WIDTH, CSS.Attribute.BORDER_BOTTOM_WIDTH, CSS.Attribute.BORDER_LEFT_WIDTH});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.CELLPADDING,
-                                 new CSS.Attribute[]{CSS.Attribute.PADDING});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.CELLSPACING,
-                                 new CSS.Attribute[]{CSS.Attribute.BORDER_SPACING});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.MARGINWIDTH,
-                                 new CSS.Attribute[]{CSS.Attribute.MARGIN_LEFT,
-                                                     CSS.Attribute.MARGIN_RIGHT});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.MARGINHEIGHT,
-                                 new CSS.Attribute[]{CSS.Attribute.MARGIN_TOP,
-                                                     CSS.Attribute.MARGIN_BOTTOM});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.HSPACE,
-                                 new CSS.Attribute[]{CSS.Attribute.PADDING_LEFT,
-                                                     CSS.Attribute.PADDING_RIGHT});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.VSPACE,
-                                 new CSS.Attribute[]{CSS.Attribute.PADDING_BOTTOM,
-                                                     CSS.Attribute.PADDING_TOP});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.FACE,
-                                 new CSS.Attribute[]{CSS.Attribute.FONT_FAMILY});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.SIZE,
-                                 new CSS.Attribute[]{CSS.Attribute.FONT_SIZE});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.VALIGN,
-                                 new CSS.Attribute[]{CSS.Attribute.VERTICAL_ALIGN});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.ALIGN,
-                                 new CSS.Attribute[]{CSS.Attribute.VERTICAL_ALIGN,
-                                                     CSS.Attribute.TEXT_ALIGN,
-                                                     CSS.Attribute.FLOAT});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.TYPE,
-                                 new CSS.Attribute[]{CSS.Attribute.LIST_STYLE_TYPE});
-        htmlAttrToCssAttrMap.put(HTML.Attribute.NOWRAP,
-                                 new CSS.Attribute[]{CSS.Attribute.WHITE_SPACE});
+        htmlAttrToCssAttrMap = Map.ofEntries(
+                Map.entry(HTML.Attribute.COLOR,
+                        new CSS.Attribute[]{CSS.Attribute.COLOR}),
```

Comment:
```suggestion
                Map.entry(HTML.Attribute.COLOR,
                          new CSS.Attribute[]{CSS.Attribute.COLOR}),
```
I'd align the wrapped line to the opening parenthesis, the difference is subtle in this case, yet it would follow the style used previously.